cpp/ql/lib/semmle/code/cpp/models/Models.qll

```diff
@@ -9,6 +9,7 @@ private import implementations.Iterator
 private import implementations.MemberFunction
 private import implementations.Memcpy
 private import implementations.Memset
+private import implementations.ODBC
```

Comment:
Maybe we should move it down to the other SQL-like imports?

Reply:
Done